presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/H2ResourceGroupsDao.java

```diff
@@ -109,9 +111,11 @@ void updateSelector(
             @Bind("user_regex") String newUserRegex,
             @Bind("source_regex") String newSourceRegex,
             @Bind("client_tags") String newClientTags,
+            @Bind("query_type") String newQueryType,
             @Bind("old_user_regex") String oldUserRegex,
             @Bind("old_source_regex") String oldSourceRegex,
-            @Bind("old_client_tags") String oldClientTags);
+            @Bind("old_client_tags") String oldClientTags,
+            @Bind("old_query_type") String oldQueryType);
```

Comment:
Do we really need this change to `updateSelector` method? I don't see any tests using this functionality other than passing simple/dummy values.

Reply:
also I don't see `old_query_type` being used in the `SQL` above, so why need it?